clippy_lints/src/write.rs

```diff
@@ -212,13 +212,15 @@ impl EarlyLintPass for Pass {
             let check_tts = check_tts(cx, &mac.node.tts, true);
             if let Some(fmtstr) = check_tts.0 {
                 if fmtstr == "" {
+                    let suggestion = check_tts.1.map_or("v", |expr| snippet(cx, expr.span, "v").into_owned().as_str());
+
                     span_lint_and_sugg(
                         cx,
                         WRITELN_EMPTY_STRING,
                         mac.span,
-                        format!("using `writeln!({}, \"\")`", check_tts.1).as_str(),
+                        format!("using writeln!({}, \"\")", suggestion).as_str(),
                         "replace it with",
-                        format!("using `writeln!({})`", check_tts.1),
+                        format!("writeln!({})", "v"),
```

Comment:
`"v"` -> `suggestion`

Reply:
How should I check for the default `v` suggestion? There are couple of cases where we have None for the Expression and not sure how to test this.